lib/infrastructure/class-wp-http-response.php

```diff
@@ -167,9 +167,7 @@ public function set_data( $data ) {
 	 *
 	 * @return mixed Any JSON-serializable value.
 	 */
-	// @codingStandardsIgnoreStart
 	public function jsonSerialize() {
-	// @codingStandardsIgnoreEnd
```

Comment:
Not since February: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/307